### ISSUE TITLE
DDP Rate Limit is active for first admin

### ISSUE THREAD

**Issue by Soldia1138** [RocketChat/Rocket.Chat#13329]
### Description:

DDP Rate limit affects first created admin user as well, breaking the setup wizard as it's failing at the last step. As one can't finish update, it's not possible to change the Rate limiter settings via Settings menu.

### Steps to reproduce:

1. Deploy a brand new instance of RocketChat (for me it was docker-compose)
2. Go to the RocketChat-Instance URL
3. Create first admin via Setup Wizard
4. Get the "too many requests" error at step 4 of setup wizart

### Expected behavior:

DDP Rate Limit not active for first user

### Actual behavior:

DDP Rate Limit active for first user

### Server Setup Information:

- Version of Rocket.Chat Server: 0.74.1
- Operating System: Ubuntu 18.04
- Deployment Method: docker-compose
- Number of Running Instances: 1

### Additional context

Downgraded to 0.74.0 and it works as expected

### Relevant logs:

Example log:
```
rocketchat_1_794719855b5c | DDP RATE LIMIT: limit by connectionId per method
rocketchat_1_794719855b5c | {
rocketchat_1_794719855b5c |   "allowed": false,
rocketchat_1_794719855b5c |   "timeToReset": 8318,
rocketchat_1_794719855b5c |   "numInvocationsLeft": 0,
rocketchat_1_794719855b5c |   "userId": "DEnHruwNddTJnD79m",
rocketchat_1_794719855b5c |   "clientAddress": "172.18.0.2",
rocketchat_1_794719855b5c |   "type": "method",
rocketchat_1_794719855b5c |   "name": "saveSetting",
rocketchat_1_794719855b5c |   "connectionId": "CbhLoe4KtFmHPZk3d"
```

**Comment by pjv** [comment]
yeah, what they said ⬆️

kinda frustrating hitting this trying to set up first rocketchat instance.

is there any workaround for getting past the setup wizard?

**Comment by itbeckham7** [comment]
i am too. 
because this error, i can't complete setup wizard.

**Comment by pjv** [comment]
my work-around was changing my docker-compose file from:

`image: rocketchat/rocket.chat:latest`

to 

`image: rocketchat/rocket.chat:0.74.0`

then i was able to complete the setup wizard.

**Comment by Soldia1138** [comment]
Same here. Starting with 0.74.0 solves the issue. upgrading to 0.74.1 after setup is complete works fine.

**Comment by rodrigok** [comment]
@Soldia1138 thanks for the feedback, we will fix this and release 0.74.2 ASAP

**Comment by 0zx08509** [comment]
Confirmed I am getting the same issue with regular installation method of 0.74.1 on RHEL 7.6:
=====
Feb  5 12:21:51 (redacted) rocketchat: DDP RATE LIMIT: limit by connectionId per method
Feb  5 12:21:51 (redacted) rocketchat: {
Feb  5 12:21:51 (redacted) rocketchat: "allowed": false,
Feb  5 12:21:51 (redacted) rocketchat: "timeToReset": 300,
Feb  5 12:21:51 (redacted) rocketchat: "numInvocationsLeft": 0,
Feb  5 12:21:51 (redacted) rocketchat: "userId": "eBcPviQrfJXd6oyJv",
Feb  5 12:21:51 (redacted) rocketchat: "clientAddress": "10.1.8.133",
Feb  5 12:21:51 (redacted) rocketchat: "type": "method",
Feb  5 12:21:51 (redacted) rocketchat: "name": "saveSetting",
Feb  5 12:21:51 (redacted) rocketchat: "connectionId": "CazSo6JvpnnsaaYWe"
Feb  5 12:21:51 (redacted) rocketchat: }
=====
